gpflow/quadrature/base.py

```diff
@@ -31,26 +31,26 @@ def __call__(self, fun, mean, var, *args, **kwargs):
         The computations broadcast along batch-dimensions, represented by [b1, b2, ..., bX].
 
         :param fun: Callable or Iterable of Callables that operates elementwise, with
-            signature f(X, *args, **kwargs). Moreover, if must satisfy the shape-mapping:
-                X shape: [b1, b2, ..., bX, N_quad_points, d],
-                    usually [N, N_quad_points, d]
-                f(X) shape: [b1, b2, ...., bf, N_quad_points, d'],
-                    usually [N, N_quad_points, 1] or [N, N_quad_points, d]
+            signature f(X, *args, **kwargs). Moreover, it must satisfy the shape-mapping:
+                X shape: [N_quad_points, b1, b2, ..., bX, d],
+                    usually [N_quad_points, N, d]
+                f(X) shape: [N_quad_points, b1, b2, ...., bf, d'],
```

Comment:
do `bf` and `bX` have to be the same? I'm assuming `d` and `d'` can be different?

Reply:
`bf` and `bX` should be the same -- I will fix it.
`d` and `d'` can be different.